package.json

```diff
@@ -541,6 +549,143 @@
                 "description": "Boolean value to indicate whether global package variables should be shown in the variables pane or not."
               }
             }
+          },
+          "attach": {
+            "required": [],
+            "properties": {
+              "processId": {
+                "type": "number",
+                "description": "The ID of the process to be debugged."
+              },
+              "mode": {
+                "enum": [
+                  "attach",
+                  "connect"
```

Comment:
Can we not re-use `debug` and `remote` here instead of introducing new modes?

Reply:
These modes match their corresponding `dlv` commands.  `attach` is new and is to attach to a process by PID.  `connect` supersedes `launch/remote`.  We could rename `connect` to `remote` here (and fix it up in the code) to lessen the churn on the end-user though.

Reply:
I would prefer renaming `connect` to `remote` so that in docs,  "remote debugging" and the `mode` being `remote` would be synonymous

Reply:
And we could print a warning in the debug console when `launch/remote` is being used, suggesting to use `attach/remote` instead